source/index.md

```diff
@@ -839,24 +839,24 @@ When using a grid map, here's how you can convert distances":
 [Converting distances]
 
 ## Trackers
-Also known as a *progress clock* in other systems. A **tracker** is just one or more circles divided into segments that the *GM* uses to track the progression of something. Every time a tracker is advanced one segment must be filled. When the circle is completely filled, the tracker is triggered and something happens. There are 3 main trackers that are always in play:
+Also known as a *progress clock* in other systems, a **tracker** is just one or more circles divided into segments that the *GM* uses to track the progression of something. Every time a tracker is advanced, one segment must be filled. When the circle is completely filled, the tracker is triggered and something happens. There are 3 main trackers that are always in play:
 
 - **Decay tracker**: Represents the toll that dungeon delving takes on the **adventurers**. Causes hunger, exhaustion and other bad things to the players. Advances every turn.
-- **Doom tracker**: There's one for each dungeon, represents the dungeon "waking up" to the **adventurers**' presence and reacting.
-- **Omens tracker**: There's one for the whole campaign. Represents the **adventurers**' terrible luck and their inevitable doom in **Blacksalt island**. Every time it gets completed something really bad happens.
+- **Doom tracker**: There's one for each dungeon, representing the dungeon "waking up" to the **adventurers**' presence and reacting.
+- **Omens tracker**: There's one for the whole campaign. Represents the **adventurers**' terrible luck and their inevitable doom in **Blacksalt island**. Every time it gets completed, something really bad happens. (???? Are these two reversed?)
```

Comment:
I wasn't sure about this - it seems like these two values are reversed, but could be completely wrong.